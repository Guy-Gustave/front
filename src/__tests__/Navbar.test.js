import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <div className="App">
        <BrowserRouter>
          <h1>Track it</h1>
        </BrowserRouter>
      </div>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
