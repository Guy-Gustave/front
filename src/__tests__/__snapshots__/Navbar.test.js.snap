// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <h1>
    Track it
  </h1>
</div>
`;
